### PR TITLE
Adding multiple endpoints support for kube-burner workloads

### PR DESCRIPTION
**Pull request by vishnuchalla**
### Description
Added support to scrape multiple prometheus endpoints as in https://github.com/cloud-bulldozer/kube-burner/pull/251

### Testing
Tested using below commands
``` 
LOG_LEVEL=debug WORKLOAD=pod-density PROM_URL=https://www.exampleurl.com METRICS_ENDPOINT=/home/vchalla/metrics endpoints.yaml ./run.sh 
```
``` 
LOG_LEVEL=debug WORKLOAD=pod-density METRICS_ENDPOINT=/home/vchalla/metrics-endpoints.yaml ./run.sh
```
```
LOG_LEVEL=debug WORKLOAD=pod-density ./run.sh
```

### CODE REVIEW

File: workloads/kube-burner/common.sh [hunk 1]
```diff
@@ -70,13 +70,20 @@ run_workload() {
   CMD="timeout ${JOB_TIMEOUT} ${KUBE_DIR}/kube-burner init --uuid=${UUID} -c $(basename ${WORKLOAD_TEMPLATE}) --log-level=${LOG_LEVEL}"
   # When metrics or alerting are enabled we have to pass the prometheus URL to the cmd
   if [[ ${INDEXING} == "true" ]] || [[ ${PLATFORM_ALERTS} == "true" ]] ; then
-    CMD+=" -u=${PROM_URL} -t ${PROM_TOKEN}"
+    if [[ -n ${PROM_URL} ]] && [[ ${PROM_URL} != "None" ]]; then
```
Comment (dry923) [inline workloads/kube-burner/common.sh:73]: If we are already in the indexing if we need either A) a prom url or B) the metrics endpoint. these two new ifs should be condensed down to 1 if/else IMHO and then document the precedence.

Comment (vishnuchalla) [reply workloads/kube-burner/common.sh]: Updated as suggested. Metrics endpoints option has precedence over prom URL. Thanks.

File: workloads/kube-burner/common.sh [hunk 2]
```diff
@@ -70,20 +70,19 @@ run_workload() {
   CMD="timeout ${JOB_TIMEOUT} ${KUBE_DIR}/kube-burner init --uuid=${UUID} -c $(basename ${WORKLOAD_TEMPLATE}) --log-level=${LOG_LEVEL}"
   # When metrics or alerting are enabled we have to pass the prometheus URL to the cmd
   if [[ ${INDEXING} == "true" ]] || [[ ${PLATFORM_ALERTS} == "true" ]] ; then
-    if [[ -n ${PROM_URL} ]] && [[ ${PROM_URL} != "None" ]]; then
-      CMD+=" -u ${PROM_URL}"
-    fi
-    CMD+=" -t ${PROM_TOKEN}"
+      if [[ -n ${METRICS_ENDPOINT} ]]; then
```
Comment (rsevilla87) [inline workloads/kube-burner/common.sh:73]: Can you add this var to the docs?

**Comment by rsevilla87** [comment]
We will need to bump the kube-burner version as well as this feature is not available in the binary version used in this repo (v0.173).

Meanwhile we can work in parallel on bumping the kube-burner version along with using the OCP wrapper so we can simplify the logic of these scripts as discussed in the tooling meeting held today. cc: @dry923 @jtaleric @paigerube14

File: workloads/kube-burner/common.sh (hunk 2 above)
Comment (vishnuchalla) [reply workloads/kube-burner/common.sh:73]: Its already present in the README.md: https://github.com/cloud-bulldozer/e2e-benchmarking/pull/537/files#diff-1bcb772094813e10febb0bc183aa8270d15743e10caceebb2b6fdfc829e6c54eR33. Let me know if there are additional places where this var needs to be added. Thanks!

**Comment by vishnuchalla** [comment]
> We will need to bump the kube-burner version as well as this feature is not available in the binary version used in this repo (v0.173).
> 
> Meanwhile we can work in parallel on bumping the kube-burner version along with using the OCP wrapper so we can simplify the logic of these scripts as discussed in the tooling meeting held today. cc: @dry923 @jtaleric @paigerube14

I have updated the kube-burner version with latest release.

**Review by dry923 — APPROVED** [review]
LGTM!

**Review by jtaleric — APPROVED** [review]
lgtm!

**Pull request merged**